Plugin/CppApi/include/ToolResourceProvider.h

```diff
@@ -82,6 +83,7 @@ public slots:
   void spatialReferenceChanged();
   void identifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
   void setMouseCursorRequested(const QCursor& cursor);
+  void screenToLocationCompleted(QUuid taskId, Point location);
```

Comment:
And then... `void screenToLocationCompleted(QUuid taskId, const Point& location);`